### PR TITLE
Added weekly summaries.

### PR DESCRIPTION
**Pull request by Elland**
We now have weekly summaries, giving users an idea of how many minutes they gained during winter. 👍

### CODE REVIEW

File: Daylight.xcodeproj/xcshareddata/xcschemes/Tests.xcscheme [hunk 1]
```diff
@@ -1,74 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (designatednerd) [inline Daylight.xcodeproj/xcshareddata/xcschemes/Tests.xcscheme]: any particular reason this scheme got deleted?

File: Library/Message.swift [hunk 2]
```diff
@@ -46,6 +48,19 @@ struct Message {
...
 
+    private let weeklySummaryMoreSunlightMessages = [
+        "The Winter Solstice becomes a distant memory, as we gained **%@** of extra sunlight this past week!",
+        "We saw **%@** of daylight during the past week, inching us further away from the dark days of winter!",
+        "Daylight time increased by **%@** the past few days. It’s allowed to start daydreaming about the long and toasty days of summer!"
+    ]
+
+    private let weeklySummaryLessSunlightMessages = [
+        "OK, daylight is dwindling. Make sure to bundle up, the sun is out there for **%@** during the coming week!",
+        "Yes, the sun drops and drops. It’s time to leave the Misery Olympics and enjoy all of the **%@** of daylight the coming days!",
```
Comment (designatednerd) [inline Library/Message.swift]: 🥇 🥈 🥉

File: Library/Notifier.swift [hunk 3]
```diff
@@ -14,35 +13,19 @@ struct Notifier {
...
-        let format = NSLocalizedString("number_of_minutes", comment: "")
-        let minuteString = String.localizedStringWithFormat(format, minutesRounded)
-        let formattedMessage = String(format: generatedMessage, minuteString)
-
-        return formattedMessage
+        return Message.notificationMessage(for: date, coordinates: location.coordinates)
     }
 
     static func cancelAllNotifications() {
-        // TODO: Fix deprecated UILocalNotification
```
Comment (designatednerd) [inline Library/Notifier.swift]: any particular reason you removed the `TODO`s on these?

**Comment by designatednerd** [comment]
@Elland also, you've got a whole mess of failing tests

File: Daylight.xcodeproj/xcshareddata/xcschemes/Tests.xcscheme (hunk 1 above)
Comment (Elland) [reply Daylight.xcodeproj/xcshareddata/xcschemes/Tests.xcscheme]: Yeah, I don't think we should use Test schemes, testing from the main scheme makes more sense to me. 👍

File: Library/Notifier.swift (hunk 3 above)
Comment (Elland) [reply Library/Notifier.swift]: Kind of a double-warning, since we already have the warning for using deprecated APIs. 👍

**Comment by Elland** [comment]
Yeah, we need to settle a few questions first about dealing with some edge cases, that will in turn change the resulting messages, requiring tests be rewritten again, so I'm postponing fixing those for now. 👍

File: Library/Notifier.swift (hunk 3 above)
Comment (designatednerd) [reply Library/Notifier.swift]: Why aren't we actually fixing the warning?

**Comment by Elland** [comment]
> In the future, could you do the Carthage stuff as its own PR? Makes it a lot easier to find the relevant code changes and have a better idea of how big a PR actually is. Thanks!

That's the plan. 👍

File: Library/Notifier.swift (hunk 3 above)
Comment (Elland) [reply Library/Notifier.swift]: That would be my next step. 👍

**Comment by Elland** [comment]
@designatednerd #110 :)

**Comment by Elland** [comment]
Cleared up Carthage changes.

**Comment by Elland** [comment]
Squashed some commits.

File: Library/Double+Helpers.swift [hunk 4]
```diff
@@ -1,10 +0,0 @@
-import Darwin
-
-extension Double {
-
-    func minuteString() -> String {
```
Comment (Elland) [inline Library/Double+Helpers.swift]: No longer used.

File: Library/Message.swift [hunk 5]
```diff
@@ -1,12 +1,14 @@
...
 import UIKit
+import SweetSwift
 
 class DateHasher {
     private var hashingDateFormatter: DateFormatter = {
         let formatter = DateFormatter()
         // Use a manually created date format.
         // Ensures strings are the same regardless of user locale.
-        formatter.dateFormat = "D MMMM yyyy"
+        formatter.dateFormat = "dd MMMM yyyy"
```
Comment (Elland) [inline Library/Message.swift]: Was using day of the year instead of day of the month. 🙈

**Comment by Elland** [comment]
Merging this now, please feel free to take a look when you're back @designatednerd 👍

File: Library/Message.swift (hunk 5 above)
Comment (designatednerd) [reply Library/Message.swift]: oopsies
